assets/js/author.js

```diff
@@ -0,0 +1,25 @@
+var $ = jQuery;
+const loadMore = $('button.load-more');
+loadMore.on('click', (e) => {
```

Comment:
please replace the arrow function with function() so we can use `this` and refer to the actual element that the event was called on.